messaging.rst

```diff
@@ -263,13 +263,14 @@ Mediated Transfer
 
 A :term:`Mediated Transfer` is a hash-time-locked transfer. Currently raiden supports only one type of lock. The lock has an amount that is being transferred, a :term:`secrethash` used to verify the secret that unlocks it, and a :term:`lock expiration` to determine its validity.
 
-Mediated transfers have an :term:`initiator` and a :term:`target` and a number of hops in between. The number of hops can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of hops a mediated transfer will require ``6N + 8`` messages to complete. These are:
+Mediated transfers have an :term:`initiator` and a :term:`target` and a number of hops in between. The number of hops can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of hops a mediated transfer will require ``10N + 14`` messages to complete. These are:
```

Comment:
We should also define `hops` or decide wether we use it at all.
We use `mediator` for the nodes who are part of the mediation path, excluding the `initiator` and `target`.
I have also heard about `hop` = the transfer between 2 nodes (either initiator->mediator, mediator->mediator, mediator->target, initiator->mediator). And the definition of a `hop` is `jump, move by jumping, pass quickly from one place to another`.

In this case, by looking at the count values, we seem to use `hop = mediator` - and it may be confusing.

Reply:
I'll remove "hop" here.

Reply:
Done!